src/Routers/SchemasRouter.js

```diff
@@ -35,7 +35,42 @@ function getOneSchema(req) {
     });
 }
 
-function createSchema(req) {
+const checkIfDefinedSchemasIsUsed = req => {
+  if (req.config && req.config.schema && req.config.schema.lockSchemas === true) {
```

Comment:
`req.config?.schema?.lockSchemas === true`

Reply:
on it